=== RUN  TestMatch

-- PASS: TestMatch

=== RUN  TestGlob
Glob(`match.go`) = []string(nil) want match.go
Glob(`mat?h.go`) = []string(nil) want match.go
Glob(`*`) = []string(nil) want match.go
Glob(`../*/match.go`) = []string(nil) want ../filepath/match.go

-- FAIL: TestGlob

=== RUN  TestCVE202230632
-- SKIP: testCVE202230632

-- PASS: TestCVE202230632

=== RUN  TestGlobError

-- PASS: TestGlobError

=== RUN  TestGlobUNC

-- PASS: TestGlobUNC

=== RUN  TestGlobSymlink
Exception: :os.symlink is not yet implemented
Called from stdgo._internal.os._Os_symlink.$Os_symlink_Fields_.symlink (stdgo/_internal/os/Os_symlink.hx line 2)
Called from _internal.path.filepath_test._Filepath_test_testglobsymlink.$Filepath_test_testglobsymlink_Fields_.testGlobSymlink (_internal/path/filepath_test/Filepath_test_testglobsymlink.hx line 24)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestGlobSymlink

=== RUN  TestWindowsGlob
skipping windows specific testException: __skip__
Called from _internal.path.filepath_test._Filepath_test_testwindowsglob.$Filepath_test_testwindowsglob_Fields_.testWindowsGlob (_internal/path/filepath_test/Filepath_test_testwindowsglob.hx line 170)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- SKIP: TestWindowsGlob

-- PASS: TestWindowsGlob

=== RUN  TestNonWindowsGlobEscape
Glob(`\match.go`) = [] want [match.go]Exception: __fail__
Called from stdgo._internal.testing.$T_common_static_extension.failNow (stdgo/_internal/testing/Testing_t_common_static_extension.hx line 94)
Called from stdgo._internal.testing.$T_common_static_extension.fatalf (stdgo/_internal/testing/Testing_t_common_static_extension.hx line 58)
Called from stdgo._internal.testing.T_.~get_fatalf.0 (stdgo/_internal/testing/Testing_t_.hx line 55)
Called from _internal.path.filepath_test._Filepath_test_testnonwindowsglobescape.$Filepath_test_testnonwindowsglobescape_Fields_.testNonWindowsGlobEscape (_internal/path/filepath_test/Filepath_test_testnonwindowsglobescape.hx line 19)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestNonWindowsGlobEscape

=== RUN  TestClean
skipping malloc count in short mode
Exception: __skip__
Called from stdgo._internal.testing.$T_common_static_extension.skipNow (stdgo/_internal/testing/Testing_t_common_static_extension.hx line 40)
Called from stdgo._internal.testing.$T_common_static_extension.skip (stdgo/_internal/testing/Testing_t_common_static_extension.hx line 52)
Called from stdgo._internal.testing.T_.~get_skip.0 (stdgo/_internal/testing/Testing_t_.hx line 80)
Called from _internal.path.filepath_test._Filepath_test_testclean.$Filepath_test_testclean_Fields_.testClean (_internal/path/filepath_test/Filepath_test_testclean.hx line 36)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- SKIP: TestClean

-- PASS: TestClean

=== RUN  TestIsLocal

-- PASS: TestIsLocal

=== RUN  TestFromAndToSlash

-- PASS: TestFromAndToSlash

=== RUN  TestSplitList

-- PASS: TestSplitList

=== RUN  TestSplit

-- PASS: TestSplit

=== RUN  TestJoin

-- PASS: TestJoin

=== RUN  TestExt

-- PASS: TestExt

=== RUN  TestWalk
unexpected errors: [Exception: SysError(Can't open testdata for update)
Called from sys.io.$File.update (/home/runner/haxe/versions/b0b2997/std/hl/_std/sys/io/File.hx line 87)
Called from stdgo._internal.os._Os_openfile.$Os_openfile_Fields_.openFile (stdgo/_internal/os/Os_openfile.hx line 8)
Called from stdgo._internal.os._Os_open.$Os_open_Fields_.open (stdgo/_internal/os/Os_open.hx line 3)
Called from stdgo._internal.path.filepath._Filepath__readdirnames.$Filepath__readdirnames_Fields_._readDirNames (stdgo/_internal/path/filepath/Filepath__readdirnames.hx line 3)
Called from stdgo._internal.path.filepath._Filepath__walk.$Filepath__walk_Fields_._walk (stdgo/_internal/path/filepath/Filepath__walk.hx line 8)
Called from stdgo._internal.path.filepath._Filepath_walk.$Filepath_walk_Fields_.walk (stdgo/_internal/path/filepath/Filepath_walk.hx line 8)
Called from _internal.path.filepath_test._Filepath_test_testwalk.$Filepath_test_testwalk_Fields_.~testWalk.0 (_internal/path/filepath_test/Filepath_test_testwalk.hx line 5)
Called from _internal.path.filepath_test._Filepath_test__testwalk.$Filepath_test__testwalk_Fields_._testWalk (_internal/path/filepath_test/Filepath_test__testwalk.hx line 41)
Called from _internal.path.filepath_test._Filepath_test_testwalk.$Filepath_test_testwalk_Fields_.testWalk (_internal/path/filepath_test/Filepath_test_testwalk.hx line 11)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)]Exception: __fail__
Called from _internal.path.filepath_test._Filepath_test__testwalk.$Filepath_test__testwalk_Fields_._testWalk (_internal/path/filepath_test/Filepath_test__testwalk.hx line 170)
Called from _internal.path.filepath_test._Filepath_test_testwalk.$Filepath_test_testwalk_Fields_.testWalk (_internal/path/filepath_test/Filepath_test_testwalk.hx line 11)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestWalk

=== RUN  TestWalkDir
unexpected errors: [Exception: SysError(Can't open testdata for update)
Called from sys.io.$File.update (/home/runner/haxe/versions/b0b2997/std/hl/_std/sys/io/File.hx line 87)
Called from stdgo._internal.os._Os_openfile.$Os_openfile_Fields_.openFile (stdgo/_internal/os/Os_openfile.hx line 8)
Called from stdgo._internal.os._Os_open.$Os_open_Fields_.open (stdgo/_internal/os/Os_open.hx line 3)
Called from stdgo._internal.path.filepath._Filepath__readdir.$Filepath__readdir_Fields_._readDir (stdgo/_internal/path/filepath/Filepath__readdir.hx line 3)
Called from stdgo._internal.path.filepath._Filepath__walkdir.$Filepath__walkdir_Fields_._walkDir (stdgo/_internal/path/filepath/Filepath__walkdir.hx line 15)
Called from stdgo._internal.path.filepath._Filepath_walkdir.$Filepath_walkdir_Fields_.walkDir (stdgo/_internal/path/filepath/Filepath_walkdir.hx line 8)
Called from _internal.path.filepath_test._Filepath_test__testwalk.$Filepath_test__testwalk_Fields_._testWalk (_internal/path/filepath_test/Filepath_test__testwalk.hx line 41)
Called from _internal.path.filepath_test._Filepath_test_testwalkdir.$Filepath_test_testwalkdir_Fields_.testWalkDir (_internal/path/filepath_test/Filepath_test_testwalkdir.hx line 4)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)]Exception: __fail__
Called from _internal.path.filepath_test._Filepath_test__testwalk.$Filepath_test__testwalk_Fields_._testWalk (_internal/path/filepath_test/Filepath_test__testwalk.hx line 170)
Called from _internal.path.filepath_test._Filepath_test_testwalkdir.$Filepath_test_testwalkdir_Fields_.testWalkDir (_internal/path/filepath_test/Filepath_test_testwalkdir.hx line 4)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestWalkDir

=== RUN  TestWalkSkipDirOnFile
Exception: :os.mkdirAll is not yet implemented
Called from stdgo._internal.os._Os_mkdirall.$Os_mkdirall_Fields_.mkdirAll (stdgo/_internal/os/Os_mkdirall.hx line 2)
Called from _internal.path.filepath_test._Filepath_test_testwalkskipdironfile.$Filepath_test_testwalkskipdironfile_Fields_.testWalkSkipDirOnFile (_internal/path/filepath_test/Filepath_test_testwalkskipdironfile.hx line 6)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestWalkSkipDirOnFile

=== RUN  TestWalkSkipAllOnFile
Exception: :os.mkdirAll is not yet implemented
Called from stdgo._internal.os._Os_mkdirall.$Os_mkdirall_Fields_.mkdirAll (stdgo/_internal/os/Os_mkdirall.hx line 2)
Called from _internal.path.filepath_test._Filepath_test_testwalkskipallonfile.$Filepath_test_testwalkskipallonfile_Fields_.testWalkSkipAllOnFile (_internal/path/filepath_test/Filepath_test_testwalkskipallonfile.hx line 6)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestWalkSkipAllOnFile

=== RUN  TestWalkFileError
Exception: :os.mkdirAll is not yet implemented
Called from _internal.path.filepath_test._Filepath_test_testwalkfileerror.$Filepath_test_testwalkfileerror_Fields_.testWalkFileError (_internal/path/filepath_test/Filepath_test_testwalkfileerror.hx line 122)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestWalkFileError

=== RUN  TestWalkSymlinkRoot
Exception: :os.mkdirAll is not yet implemented
Called from stdgo._internal.os._Os_mkdirall.$Os_mkdirall_Fields_.mkdirAll (stdgo/_internal/os/Os_mkdirall.hx line 2)
Called from _internal.path.filepath_test._Filepath_test_testwalksymlinkroot.$Filepath_test_testwalksymlinkroot_Fields_.testWalkSymlinkRoot (_internal/path/filepath_test/Filepath_test_testwalksymlinkroot.hx line 9)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestWalkSymlinkRoot

=== RUN  TestBase

-- PASS: TestBase

=== RUN  TestDir

-- PASS: TestDir

=== RUN  TestIsAbs

-- PASS: TestIsAbs

=== RUN  TestEvalSymlinks
Exception: :os.symlink is not yet implemented
Called from stdgo._internal.os._Os_symlink.$Os_symlink_Fields_.symlink (stdgo/_internal/os/Os_symlink.hx line 2)
Called from _internal.path.filepath_test._Filepath_test_testevalsymlinks.$Filepath_test_testevalsymlinks_Fields_.testEvalSymlinks (_internal/path/filepath_test/Filepath_test_testevalsymlinks.hx line 25)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestEvalSymlinks

=== RUN  TestEvalSymlinksIsNotExist
Exception: :os.isNotExist is not yet implemented
Called from _internal.path.filepath_test._Filepath_test_testevalsymlinksisnotexist.$Filepath_test_testevalsymlinksisnotexist_Fields_.testEvalSymlinksIsNotExist (_internal/path/filepath_test/Filepath_test_testevalsymlinksisnotexist.hx line 85)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestEvalSymlinksIsNotExist

=== RUN  TestIssue13582
Exception: :os.symlink is not yet implemented
Called from stdgo._internal.os._Os_symlink.$Os_symlink_Fields_.symlink (stdgo/_internal/os/Os_symlink.hx line 2)
Called from _internal.path.filepath_test._Filepath_test_testissue13582.$Filepath_test_testissue13582_Fields_.testIssue13582 (_internal/path/filepath_test/Filepath_test_testissue13582.hx line 14)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)

-- FAIL: TestIssue13582

=== RUN  TestRelativeSymlinkToAbsolute
Exception: :os.symlink is not yet implemented
Called from stdgo._internal.os._Os_symlink.$Os_symlink_Fields_.symlink (stdgo/_internal/os/Os_symlink.hx line 2)
Called from _internal.path.filepath_test._Filepath_test_testrelativesymlinktoabsolute.$Filepath_test_testrelativesymlinktoabsolute_Fields_.testRelativeSymlinkToAbsolute (_internal/path/filepath_test/Filepath_test_testrelativesymlinktoabsolute.hx line 10)
Called from stdgo._internal.testing.$M_static_extension.run (stdgo/_internal/testing/Testing_m_static_extension.hx line 33)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main (_internal/path/filepathdottest/Filepathdottest.hx line 51)
Uncaught exception: SysError(Can't delete directory tmp_8pv3f4upwk)
Called from sys.$FileSystem.deleteDirectory(/home/runner/haxe/versions/b0b2997/std/hl/_std/sys/FileSystem.hx:92)
Called from stdgo._internal.os._Os_removeall.$Os_removeall_Fields_.removeAll(stdgo/_internal/os/Os_removeall.hx:25)
Called from stdgo._internal.testing.$T_common_static_extension.~tempDir.1(stdgo/_internal/testing/Testing_t_common_static_extension.hx:20)
Called from stdgo._internal.testing.$M_static_extension.run(stdgo/_internal/testing/Testing_m_static_extension.hx:41)
Called from _internal.path.filepathdottest._Filepathdottest.$Filepathdottest_Fields_.main(_internal/path/filepathdottest/Filepathdottest.hx:51)
Called from .init(?:1)
